scripts/consume_sitemaps.py

```diff
@@ -49,11 +53,12 @@ def main(ctx, url: str, authority: str, ignore_last_modified: bool):
         pool_connections=CONCURRENT_DOWNLOADS * 2, pool_maxsize=CONCURRENT_DOWNLOADS * 2
     )
     rsession.mount("http://", adapter)
+    rsession.mount("https://", adapter)
     db_url = isb_web.config.Settings().database_url
     db_session = SQLModelDAO(db_url).get_session()
     if authority is not None:
         authority = authority.upper()
-    isb_lib.core.things_main(ctx, db_url, solr_url, "INFO", False)
+    isb_lib.core.things_main(ctx, db_url, solr_url, "CRITICAL", False)
```

Comment:
Turn down the logging as the chattiness was costing a few seconds per thousand records.